src/css/tailwind.css

```diff
@@ -47,6 +47,42 @@ path.logo-nor {
   fill: theme("colors.gray.800");
 }
 
+@media (prefers-color-scheme: dark) {
```

Comment:
These styles are duplicated from above which isn't ideal, if you have any ideas about how they could be kept in sync I'd love to hear them! I'm not familiar with the CSS setup you're running here